.github/workflows/react-lint-testing.yml

```diff
@@ -10,28 +10,23 @@ on:
         type: boolean
 
 jobs:
-  test:
+  build:
     runs-on: ubuntu-20.04
-
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-
       - name: Set Node.js 16.x
         uses: actions/setup-node@v2
         with:
           node-version: 16.x
-
       - name: Install dependencies
-        run: |
-          npm ci
-
+        run: npm ci
       - name: Run lint
-        run: |
-          npm run lint
-
+        run: npm run lint
+      - name: Run component tests
```

Comment:
could add integration tests to `release.yml` as well (I'm just copying what you did on `holder`). Also README how to run tests